client/components/systems/eote/character.vue

```diff
@@ -23,6 +23,7 @@
                 <experience class="flex-shrink-1 flex-grow-0" :readonly="!isAuthorized"></experience>
                 <defenses class="mt-1 flex-shrink-1 flex-grow-0" :readonly="!isAuthorized"></defenses>
                 <wounds class=" mt-1 flex-shrink-1 flex-grow-0" :readonly="!isAuthorized"></wounds>
+                <force-pool v-if="mode === 'eote'" class=" mt-1 flex-shrink-1 flex-grow-0" :readonly="!isAuthorized"></force-pool>
```

Comment:
I’m just missing context here. Based on the file name this is already eote, so I don’t know why we’d need an if to check the mode here.

Reply:
The same file is used for genesys as well. Hence the need for the check.